From 1e68f191fc3812e64b053598a87e96d24d218657 Mon Sep 17 00:00:00 2001
From: Vicentiu Galanopulo <vicentiu@balena.io>
Date: Mon, 22 Feb 2021 12:55:57 +0100
Subject: [PATCH] Increase default env size

Upstream-Status: Inappropriate [configuration]
Signed-off-by: Vicentiu Galanopulo  <vicentiu@balena.io>
---
 include/configs/iot-gate-imx8.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/configs/iot-gate-imx8.h b/include/configs/iot-gate-imx8.h
index 103fc164c7..6486abbdfb 100644
--- a/include/configs/iot-gate-imx8.h
+++ b/include/configs/iot-gate-imx8.h
@@ -17,6 +17,8 @@
 #undef CONFIG_EXTRA_ENV_SETTINGS
 #undef CONFIG_BOOTCOMMAND
 
+#define CONFIG_ENV_SIZE 0x3000
+
 #define ENV_CONSOLE console=ttymxc2,115200 earlycon=ec_imx6q,0x30880000,115200
 #define CONFIG_EXTRA_ENV_SETTINGS		\
 	CONFIG_MFG_ENV_SETTINGS \
-- 
2.17.1

